app/services/funnel/registration/add_mfa.rb

```diff
@@ -3,14 +3,21 @@
 module Funnel
   module Registration
     class AddMfa
-      def self.call(user_id, mfa_method, analytics)
+      def self.call(user_id, mfa_method, analytics, threatmetrix_attrs)
         now = Time.zone.now
         funnel = RegistrationLog.create_or_find_by(user_id: user_id)
         return if funnel.registered_at.present?
 
         analytics.user_registration_user_fully_registered(mfa_method: mfa_method)
+        process_threatmetrix_for_user(
+          threatmetrix_attrs,
+        )
         funnel.update!(registered_at: now)
       end
+
+      def self.process_threatmetrix_for_user(threatmetrix_attrs)
+        AccountCreationThreatMetrixJob.perform_later(**threatmetrix_attrs)
```

Comment:
This might be a problem in 50/50 state: https://handbook.login.gov/articles/manage-50-50-state.html#add-a-new-job

Maybe we can get around it by feature-flagging an early return?